pkg/virt-controller/watch/vm.go

```diff
@@ -941,6 +1004,9 @@ func (c *VMController) updateStatus(vm *virtv1.VirtualMachine, vmi *virtv1.Virtu
 	// Set created and ready flags
 	vm.Status.Created = created
 	vm.Status.Ready = ready
+	if clearRestartRequest {
```

Comment:
Actually, we'll always need this boolean. In any case where a stop/start/restart/whatever is being attempted and fails (thus the VM gets re-enqueued), it would be inappropriate to clear this flag would it not?